addons/auth_oauth/data/auth_oauth_data.xml

```diff
@@ -6,7 +6,7 @@
             <field name="auth_endpoint">https://accounts.odoo.com/oauth2/auth</field>
             <field name="scope">userinfo</field>
             <field name="validation_endpoint">https://accounts.odoo.com/oauth2/tokeninfo</field>
-            <field name="css_class">fa fa-fw o_custom_icon</field>
+            <field name="css_class">oi oi-odoo</field>
```

Comment:
Good catch for the upgrade script adaptation, I did not even know `update_record_from_xml` existed, I though it was going to be annoying :+1: